awkward-cpp/pyproject.toml

```diff
@@ -56,6 +56,9 @@ Chat = "https://gitter.im/Scikit-HEP/awkward-array"
 Releases = "https://github.com/scikit-hep/awkward-1.0/releases"
 
 
+[tool.scikit-build]
+minimum-version = "0.0"
```

Comment:
Thanks, @agoose77!

Reply:
I'd also recommend `sdist.reproducible = true`, just to clarify you expect that. If the default changes, the `minimum-version` will protect you, but might as well be explicit that that's critical to your use.